src/index.ts

```diff
@@ -1 +1,232 @@
-export {};
+import { decompress as decompressZstd } from '@mongodb-js/zstd';
+import { satisfies as pep440Satisfies } from '@renovatebot/pep440';
+import { createHash } from 'crypto';
+import decompress from 'decompress';
+import type { Options as ExecaOptions } from 'execa';
+import { execa } from 'execa';
+import fse from 'fs-extra';
+import globalCacheDir from 'global-cache-dir';
+import { Octokit } from 'octokit';
+import { join } from 'path';
+import semverCompare from 'semver/functions/compare.js';
+import semverSatifies from 'semver/functions/satisfies.js';
+import { version as autopyVersion } from '../package.json';
+
+/**
+ * A semver version or range that specifies the desired Python version to download or use for a virtual environment. The
+ * version or range should follow the format defined by [semver](https://www.npmjs.com/package/semver). As
+ * [python-build-standalone](https://github.com/indygreg/python-build-standalone) doesn't offer all Python versions, one
+ * should typically specify a range (usually a tilde or caret range). If not specified, the latest available version is
+ * used.
+ */
+export type SemverVersionSpecifier = string | undefined;
+
+const getPythonDownloadLink = async (versionRange: SemverVersionSpecifier) => {
+    const version = versionRange ?? '>= 0';
+
+    const targetMap = {
+        darwin: {
+            x64: 'x86_64-apple-darwin',
+            arm64: 'aarch64-apple-darwin',
+        },
+        win32: {
+            x64: 'x86_64-pc-windows-msvc-shared',
+            ia32: 'i686-pc-windows-msvc-shared',
+        },
+        linux: {
+            x64: 'x86_64-unknown-linux-gnu',
+            ia32: 'i686-unknown-linux-gnu',
+            arm64: 'aarch64-unknown-linux-gnu',
+        },
+    };
+    const target = targetMap[process.platform as 'win32'][process.arch as 'x64'];
+    if (!target) throw new Error(`Unsupported platform: ${process.platform}-${process.arch}`);
+
+    const octokit = new Octokit({
+        userAgent: `autopy/${autopyVersion} (https://github.com/tweaselORG/autopy)`,
+    });
+
+    const releasesIterator = octokit.paginate.iterator(octokit.rest.repos.listReleases, {
+        owner: 'indygreg',
+        repo: 'python-build-standalone',
+        // eslint-disable-next-line camelcase
+        per_page: versionRange === undefined ? 1 : 10,
+    });
+
+    for await (const { data: releases } of releasesIterator) {
+        for (const release of releases) {
+            if (release.draft) continue;
```

Comment:
`draft` !== `prerelease` (https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release):

![image](https://github.com/tweaselORG/autopy/assets/4048809/7b986920-f997-45dd-a03d-b04b7d8e04c3)

So, we don't really need the check at all tbh, since we're not authenticating.